### ISSUE TITLE
Use case (end-user): View a map in fullscreen mode

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#109]
Use-case of a user wanting to view the entire map in full-screen mode.
https://fullscreen.spec.whatwg.org/

**Comment by AmeliaBR** [comment]
I agree that this is a distinct use case from viewing map data on its own, extracted from the web page.  This is more about expanding the map viewer widget within its current web page (as is standard for video viewers). If anything, I'd say this is closer to the picture-in-picture use case (#107), since the APIs for the two are similar.

**Comment by Malvoz** [comment]
I think it's unfortunate that only 1 out of the 12 web map implementations of a [basic web map](https://maps4html.org/HTML-Map-Element-UseCases-Requirements/examples/create-map.html) displays a fullscreen control to the user. IMO that's a usability/accessibility issue, especially on mobile devices.
